zingolib/src/lightclient/describe.rs

```diff
@@ -303,7 +302,7 @@ impl LightClient {
         let mut value_transfers: Vec<ValueTransfer> = Vec::new();
         let transaction_summaries = self.transaction_summaries().await;
 
-        for tx in transaction_summaries.iter() {
```

Comment:
Can we make this into an argument that the command takes, that optionally reverses it? I would like to be able to keep earliest-first order in my zingocli usage.